src/providers/wfs/qgswfsdatasourceuri.cpp

```diff
@@ -373,6 +373,12 @@ bool QgsWFSDataSourceURI::hideDownloadProgressDialog() const
   return mURI.hasParam( QgsWFSConstants::URI_PARAM_HIDEDOWNLOADPROGRESSDIALOG );
 }
 
+
+bool QgsWFSDataSourceURI::preferCoordinatesForWfst11() const
```

Comment:
This doesn't seem to be used at all. I guess there will be a forth coming commit that will make use of it instead of the currently disabled GML3 code path ?

Reply:
A broken cherry pick from another branch, I'll fix it now.